crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/controller/CrownProceedingControllerTest.java

```diff
@@ -157,9 +157,9 @@ void givenAInValidInput_whenUpdateIsInvoked_RequestObjectFailsValidation() throw
 
     @Test
     void givenAnInput_whenUpdateIsInvokedAndValidationFails_thenBadRequestResponseIsReturned() throws Exception {
-        var apiUpdateApplicationRequest =
-                TestModelDataBuilder.getApiUpdateApplicationRequest(IS_VALID);
-        var updateApplicationRequestJson = objectMapper.writeValueAsString(apiUpdateApplicationRequest);
+        var apiUpdateCrownCourtRequest =
+                TestModelDataBuilder.getApiUpdateCrownCourtRequest(IS_VALID);
+        var updateApplicationRequestJson = objectMapper.writeValueAsString(apiUpdateCrownCourtRequest);
         when(crownCourtDetailsValidator.checkCCDetails(any())).thenThrow(new ValidationException("Cannot have Crown Court outcome without Mags Court outcome"));
         mvc.perform(RequestBuilderUtils.buildRequestGivenContent(
                         HttpMethod.PUT, updateApplicationRequestJson, ENDPOINT_URL + "/update-crown-court"))
```

Comment:
Can we make this a constant? I also think we should be reusing the error message constant from the `CrownCourtDetailsValidator`